File: codewars/Simple Pig Latin/simple_pig_latin.js
function pigIt(str){
    let res = [];
  
    res = str.split(' ')
    for (i = 0; i < res.length; i++) {
      if (res[i] === '!' || res[i] === '?') {
        break
      }
      let resStr = res[i]
      let resArr = []
      resArr = resStr.split('')
      resArr.push(resArr.shift(),'ay')
      res[i] = resArr.join('')
    }
    return res.join(' ')
  }

File: codewars/Simple Pig Latin/SimplePigLatin.py
import string

def pig_it(text):
    def transform_word(word):
        if word not in string.punctuation:
            return word[1:] + word[0] + 'ay'
        return word
    
    words = text.split()
    transformed_words = [transform_word(word) for word in words]
    return ' '.join(transformed_words)
